banzai/dark.py

```diff
@@ -41,9 +43,14 @@ def calibration_type(self):
 
     def apply_master_calibration(self, image, master_calibration_image):
         master_calibration_image *= image.exptime
+        temp_scaling_factor = np.exp(master_calibration_image.dark_temperature_coefficient * \
```

Comment:
The @property annotation for dark_temperatuer_coefficient(self) maps the function to a read of the variable?